src/redux/slices/debug-options-slice.ts

```diff
@@ -0,0 +1,89 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import {
+  DebugOptions,
+  TileColoredBy,
+  BoundingVolumeColoredBy,
+  BoundingVolumeType,
+} from "../../types";
+import { RootState } from "../store";
+
+// Define a type for the slice state
+export interface debugOptionsState {
+  value: DebugOptions;
+}
+const initialState: debugOptionsState = {
+  value: {
```

Comment:
Why do we need `value` wrapper? Can we move options to the state root. I guess it is better for performance. If we change one option it will update all selectors.